### ISSUE TITLE
GDAS and ERA5 data preparation for pytorch dataloader and finalizing graph model structure

### ISSUE THREAD

**Issue by sadeghitabas** [NOAA-EMC/ML4BC#6]
This issue address the process of data preparation steps including: data transformation and reshaping, filling missing values (if needed), normalizing, splitting to different sets and making the DataLoader to be iterated through it during the training. Also in this issue we finalize the model structure to consider Graph neural networks as an encoder-processor-decoder considering architecture suggested by Ryan Keisler (2022).

**Comment by junwang-noaa** [comment]
The work is done, close the issue.
